components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -15,6 +15,69 @@
   overflow: hidden;
 }
 
+.dragDetectionOverlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: transparent;
+  z-index: 9999; // Higher than iframe to catch drag events
+  pointer-events: auto; // Allow drag events
+}
+
+.dragOverlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  pointer-events: none;
+  z-index: 10000;
+}
+
+.dragOverlayContent {
+  background: var(--leo-color-container-background);
+  border-radius: var(--leo-radius-xl);
+  padding: var(--leo-spacing-3xl);
+  box-shadow: var(--leo-effect-elevation-03);
+  text-align: center;
+  width: min(320px, calc(100vw - 2rem));
+  max-width: 320px;
```

Comment:
do we need to specify `max-width` here?

Reply:
Fixed in 352317b766773925161f8c6404f72eb831446c5c